enterprise/j2ee.weblogic9/src/org/netbeans/modules/j2ee/weblogic9/ui/nodes/WLManagerNode.java

```diff
@@ -209,7 +209,7 @@ public void setValue(Object value) {
                    ) {
                        public Object getValue() {
                            String password = deploymentManager.getInstanceProperties().getProperty(InstanceProperties.PASSWORD_ATTR);
-                           return password.replaceAll(".", "\\*");
+                           return password.replace(".", "\\*");
```

Comment:
This is not the same as the original, right?

Reply:
Correct. Before, any character would get converted into "\\*". Now, only "." characters are converted into "\\*". Do you consider this an incorrect modification?

Reply:
I think it is fairly obvious the new code is wrong. For a password "password", before, the resulting text was "********", but now is "password". I don't see a reason why it would be desirable to replace '.' in passwords with '*', but I see obvious reasons to replace all characters of a password with '*'. (You could argue that the code could be written in a different way, and that's of course true, but changing replaceAll to replace here is not in a good direction.)

Reply:
My mistake. After I had replied, I did realize what the replacement was meant to do in the context of obscuring passwords. This change was reverted to its original replacelAll() in a later commit